docs/content/latest/migrate/migrate-from-postgresql/migrate-schema.md

```diff
@@ -163,14 +163,14 @@ CREATE TABLE contacts (
 );
 ```
 
-## Using `ysql_dump`
+## Use `ysql_dump`
 
-The PostgreSQL utility, pg_dump, can be used to dump the schema of a database. However, this schema would need  to be slightly modified as described below. 
+The PostgreSQL utility `pg_dump` can be used to dump the schema of a database.
 
-The [`ysql_dump`](../../../admin/ysql-dump) tool (a YugabyteDB-specific version of the `pg_dump` tool) can connect to an existing PostgreSQL database and export a YugabyteDB-friendly version of the schema and, therefore, includes some of the schema modifications described below. The other changes mentioned below would need to be manually performed since they depend on the use case.
+The [`ysql_dump`](../../../admin/ysql-dump) tool (a YugabyteDB-specific version of the `pg_dump` tool) can connect to an existing PostgreSQL database and export a YugabyteDB-friendly version of the schema and, therefore, includes some of the schema modifications. Additional changes might need to be performed manually, depending on the use case.
```

Comment:
Feel like using "additional" here may make it seem like it introduces new changes (outside of the schema changes mentioned above). Maybe "remaining" is the right word here?

Reply:
Changed